hw/ip/spi_device/data/spi_device_testplan.hjson

```diff
@@ -70,9 +70,34 @@
       tests: ["spi_device_dummy_item_extra_dly"]
     }
     {
-      name: async_fifo_reset
-      desc: '''Reset async fifo when SPI interface is idle
-            TODO: fifo may be fetching data from SRAM? What is the actual usage?'''
+      name: tx_async_fifo_reset
```

Comment:
@eunchan @kosta-kojdic please let me know if these 2 entries for async_fifo_reset make sense

Reply:
Looks good

Reply:
Looks good, @weicaiyang thank you very much for detailed descriptions of test flows for both tx and rx